### PR TITLE
Correctly use dbus signals for handling battery update

### PR DESCRIPTION
**Pull request by GuillaumeCisco**
Should correctly fix: https://github.com/joaorb64/joycond-cemuhook/pull/64

### CODE REVIEW

File: joycond-cemuhook.py [hunk 1]
```diff
@@ -787,6 +782,7 @@ def main():
     def signal_handler(signal, frame):
         print("Stopping servers...")
         stop_event.set()
+        loop.quit()
```
Comment (GuillaumeCisco) [inline joycond-cemuhook.py:785]: should we really quit here?

File: joycond-cemuhook.py [hunk 2]
```diff
@@ -743,6 +737,7 @@ def print_slots():
         if active_devices != taken_slots():
             print_slots()
 
+        loop.run()
```
Comment (GuillaumeCisco) [inline joycond-cemuhook.py:740]: where to put this loop run?
It blocks the other loop...

Comment (GuillaumeCisco) [reply joycond-cemuhook.py:740]: Was looking for asyncio_glib thanks to https://blogs.gnome.org/jamesh/2019/08/05/glib-integration-for-python-asyncio/
But it loop has to be declared in main thread.
It breaks adding a loop in each worker thread...

How to deal with this?

Comment (GuillaumeCisco) [reply joycond-cemuhook.py:740]: we might need to use https://github.com/altdesktop/python-dbus-next
which support asyncio with dbus

Comment (GuillaumeCisco) [reply joycond-cemuhook.py:740]: ok using a simple thread makes it work...

**Comment by joaorb64** [comment]
After such a long time, I've finally added your changes to the code. However, I ended up doing it manually. Thanks for the contribution!

**Pull request closed without merging**